core/templates/dev/head/base_components/header.html

```diff
@@ -0,0 +1,18 @@
+<%= htmlWebpackPlugin.options.meta.toHTML() %>
+<meta charset="UTF-8">
+<meta name="referrer" content="no-referrer">
+<meta name="viewport" content="width=device-width,initial-scale=1,user-scalable=yes">
+<meta name="application-name" content="<[ siteName ]>">
```

Comment:
In our codebase we don't add spaces after `<[` and before `]>`, please remove them in this file.

Reply:
Done